lib/danger/request_sources/github/github.rb

```diff
@@ -115,18 +115,20 @@ def review
 
       def setup_danger_branches
         # we can use a github specific feature here:
+        base_repo = self.pr_json["base"]["repo"]["clone_url"]
         base_branch = self.pr_json["base"]["ref"]
         base_commit = self.pr_json["base"]["sha"]
+        head_repo = self.pr_json["head"]["repo"]["clone_url"]
         head_branch = self.pr_json["head"]["ref"]
         head_commit = self.pr_json["head"]["sha"]
 
         # Next, we want to ensure that we have a version of the current branch at a known location
-        scm.ensure_commitish_exists_on_branch! base_branch, base_commit
+        scm.ensure_commitish_exists_on_branch! base_branch, base_commit, :remote => base_repo
```

Comment:
```suggestion
        scm.ensure_commitish_exists_on_branch! base_branch, base_commit, remote: base_repo
```

Reply:
I guess I'm too used to working on the OpenStreetMap code that still uses the old style. Now fixed.